### ISSUE TITLE
Deprecation in jwt.decode causes error

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/cgbeacon2#116]
Captured by one test (tests/utils/test_auth.py::test_decode_passport)

```
        # When a POST request with a valid token is sent
        headers = copy.deepcopy(HEADERS)
        headers["Authorization"] = "Bearer " + test_token
        mock_app.config["ELIXIR_OAUTH2"]["userinfo"] = mock_oauth2["userinfo"]
        mock_app.config["ELIXIR_OAUTH2"][
            "bona_fide_requirements"
        ] = "https://doi.org/10.1038/s41431-018-0219-y"
    
        response = mock_app.test_client().post(
            "/apiv1.0/query?", headers=headers, data=json.dumps(basic_query)
        )
    
        # it should return a valid response
>       assert response.status_code == 200
E       assert 403 == 200
E        +  where 403 = <Response streamed [403 FORBIDDEN]>.status_code

tests/server/blueprints/api_v1/test_views_query_auth.py:441: AssertionError
---------------------------- Captured stderr setup -----------------------------
INFO:cgbeacon2.server:Starting app envirironmental variable CGBEACON2_CONFIG
INFO:cgbeacon2.server:database connection info:Database(MongoClient(host=['127.0.0.1:27017'], document_class=dict, tz_aware=False, connect=True), 'cgbeacon2-test')
------------------------------ Captured log setup ------------------------------
INFO     cgbeacon2.server:__init__.py:21 Starting app envirironmental variable CGBEACON2_CONFIG
INFO     cgbeacon2.server:__init__.py:45 database connection info:Database(MongoClient(host=['127.0.0.1:27017'], document_class=dict, tz_aware=False, connect=True), 'cgbeacon2-test')
----------------------------- Captured stderr call -----------------------------
INFO:cgbeacon2.utils.auth:Auth Token validated.
INFO:cgbeacon2.utils.auth:Identified as someone@somewhere.se user by https://login.elixir-czech.org/oidc/.
------------------------------ Captured log call -------------------------------
INFO     cgbeacon2.utils.auth:auth.py:75 Auth Token validated.
INFO     cgbeacon2.utils.auth:auth.py:76 Identified as someone@somewhere.se user by https://login.elixir-czech.org/oidc/.
_____________________________ test_decode_passport _____________________________

test_token = 'eyJqa3UiOiJodHRwOi8vc2NpbGlmZWxhYi5zZS9qa3ciLCJraWQiOiIwMThjMGFlNS00ZDliLTQ3MWItYmZkNi1lZWYzMTRiYzcwMzciLCJhbGciOiJIU...9tZW9uZUBzb21ld2hlcmUuc2UiLCJzY29wZSI6Im9wZW5pZCBnYTRnaF9wYXNzcG9ydF92MSJ9.qe7QyaJYqOuG3DHIjNpvOX5sz0F6FKWtJpPG-yVw0C8'
header = {'alg': 'HS256', 'jku': 'http://scilifelab.se/jkw', 'kid': '018c0ae5-4d9b-471b-bfd6-eef314bc7037', 'typ': 'JWT'}
payload = {'aud': 'audience', 'exp': 1609841438, 'iss': 'https://login.elixir-czech.org/oidc/', 'scope': 'openid ga4gh_passport_v1', ...}

    def test_decode_passport(test_token, header, payload):
        """Test the function that extracts header and payload from a Base64url encoded token"""
    
>       decode_header, decoded_payload = decode_passport(test_token)

tests/utils/test_auth.py:30: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
cgbeacon2/utils/auth.py:286: in decode_passport
    payload = jwt.decode(encoded, verify=False)
/opt/hostedtoolcache/Python/3.8.6/x64/lib/python3.8/site-packages/jwt/api_jwt.py:113: in decode
    decoded = self.decode_complete(jwt, key, algorithms, options, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <jwt.api_jwt.PyJWT object at 0x7f0999a9c250>
jwt = 'eyJqa3UiOiJodHRwOi8vc2NpbGlmZWxhYi5zZS9qa3ciLCJraWQiOiIwMThjMGFlNS00ZDliLTQ3MWItYmZkNi1lZWYzMTRiYzcwMzciLCJhbGciOiJIU...9tZW9uZUBzb21ld2hlcmUuc2UiLCJzY29wZSI6Im9wZW5pZCBnYTRnaF9wYXNzcG9ydF92MSJ9.qe7QyaJYqOuG3DHIjNpvOX5sz0F6FKWtJpPG-yVw0C8'
key = '', algorithms = None, options = {'verify_signature': True}
kwargs = {'verify': False}

    def decode_complete(
        self,
        jwt: str,
        key: str = "",
        algorithms: List[str] = None,
        options: Dict = None,
        **kwargs,
    ) -> Dict[str, Any]:
        if options is None:
            options = {"verify_signature": True}
        else:
            options.setdefault("verify_signature", True)
    
        if options["verify_signature"] and not algorithms:
>           raise DecodeError(
                'It is required that you pass in a value for the "algorithms" argument when calling decode().'
            )
E           jwt.exceptions.DecodeError: It is required that you pass in a value for the "algorithms" argument when calling decode().
```

**Comment by northwestwitch** [comment]
For reference: https://github.com/cloud-gov/cg-django-uaa/issues/51

**Comment by northwestwitch** [comment]
Fixed in #117
